tools/link.py

```diff
@@ -2001,6 +2001,11 @@ def run_embind_gen(wasm_target, js_syms, extra_settings, linker_inputs):
   setup_environment_settings()
   # Use a separate Wasm file so the JS does not need to be modified after emscripten.emscript.
   settings.SINGLE_FILE = False
+  if settings.JSPI:
+    # JSPI is not needed to generate the definitions.
+    # TODO: when the emsdk node version supports JSPI, it probably makes more sense
+    # to enable it in node than disabling the setting here.
+    settings.ASYNCIFY = 0
```

Comment:
Why not `settings.JSPI = 0` here?

Reply:
JSPI really is just `ASYNCIFY = 2`. I guess it's probably more clear if only use the asyncify setting, e.g. `if settings.ASYNCIFY == 2:`.

Reply:
sgtm